backend/handlers/event.go

```diff
@@ -43,9 +45,104 @@ func (h *EventHandler) GetMany(ctx *fiber.Ctx) error {
 }
 
 func (h *EventHandler) GetOne(ctx *fiber.Ctx) error {
-	return nil
+	eventId := ctx.Params("eventId")
+
+	context, cancel := context.WithTimeout(context.Background(), time.Duration(5*time.Second))
+	defer cancel()
+
+	event, err := h.repository.GetOne(context, eventId)
+
+	if err != nil {
+		return ctx.Status(fiber.StatusBadRequest).JSON(&fiber.Map{
+			"status":  "fail/getone/statusbadrequest",
+			"message": err.Error(),
+		})
+	}
+
+	return ctx.Status(fiber.StatusOK).JSON(&fiber.Map{
+		"status":  "success",
+		"message": "",
+		"data":    event,
+	})
```

Comment:
Besides, you should create a new type representing the request (if any) and response for each handler.